tests/test_languages.py

```diff
@@ -1455,8 +1455,8 @@ def test_translation(self, shortname, datetime_string, expected_translation):
         param('nmg', "nakugú", "1 day ago"),
         param('nmg', "namáná", "in 1 day"),
         # nn
-        param('nn', "for 5 minutter siden", "5 minute ago"),
-        param('nn', "om 3 uker", "in 3 week"),
+        param('nn', "for 5 min sidan", "5 minute ago"),
+        param('nn', "om 3 veke", "in 3 week"),
```

Comment:
Norwegian Nynorsk: Can't be verified but new CLDR 39 data updates this. Google translate has no support for `Nynorsk`